### PR TITLE
Update LJSpeech preprocessing

### PR DESCRIPTION
**Pull request by Oktai15**
Signed-off-by: Oktai Tatanov <oktai.tatanov@gmail.com>

### CODE REVIEW

File: scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py [hunk 1]
```diff
@@ -22,48 +22,78 @@
...
-    # NeMo parser for text normalization
-    text_parser = parsers.make_parser(name='en')
+    if args.normalizer_class == "ENCharParser":
+        normalizer_call = parsers.make_parser(name='en')._normalize
+    elif args.normalizer_class == "Normalizer":
+        if args.whitelist_path is None:
+            raise ValueError(
+                "whitelist_path is required when using Normalizer. "
+                "You can download the whitelist for LJSpeech from https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/text_normalization/en/data/whitelist_lj_speech.tsv."
+            )
```
Comment (blisc) [inline scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py:55]: Why not just automatically download it if not specified and save it to a cache dir?

File: scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py [hunk 2]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Creates the manifest and .txt transcript files for an LJSpeech split.
-The manifest will be used for training, and the .txt files are for the MFA library to find.
+This script creates NeMo manifests which can be used for training models on LJSpeech (split is taken from https://github.com/NVIDIA/tacotron2).
+It optionally saves transcripts in .txt files (can be used for extracting durations via MFA library).
 """
```
Comment (blisc) [inline scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py:17]: Let's call your new download script in this script as well if LJSpeech is not found.

**Review by blisc — COMMENTED** [review]
Minor feedback

File: scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py (hunk 2 above)
Comment (Oktai15) [reply scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py]: Done.

File: scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py (hunk 1 above)
Comment (Oktai15) [reply scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py]: Done.

**Review by blisc — APPROVED** [review]

**Pull request merged**